src/frontend/hooks/use-resource-edit.ts

```diff
@@ -69,11 +69,11 @@ const useResourceEdit = (
           setLoading(false)
         }
       })
-      .catch(() => {
+      .catch((error) => {
         setLoading(false)
         onNotice({
-          message:
-            'There was an error updating record, Check out console to see more information.',
+          message: error.response?.data.notice
```

Comment:
in the latest release I added i18n, so all errors goes throught tranlsateMessage function().